### PR TITLE
[YUNIKORN-1410] Update user/group regex values in the defaults & test code

### PR DESCRIPTION
**Pull request by pbacsko**
### What is this PR for?
Set proper regexp values for certain variables (external user, external groups, system user).

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1410

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: pkg/plugin/admissioncontrollers/webhook/admission_controller_test.go [hunk 1]
```diff
@@ -364,9 +364,9 @@ func prepareController(t *testing.T, url string, processNs string, bypassNs stri
 		conf.AMFilteringNoLabelNamespaces:     noLabelNs,
 		conf.AMAccessControlBypassAuth:        fmt.Sprintf("%t", bypassAuth),
 		conf.AMAccessControlTrustControllers:  fmt.Sprintf("%t", trustControllers),
-		conf.AMAccessControlSystemUsers:       "system:serviceaccount:kube-system:job-controller",
-		conf.AMAccessControlExternalUsers:     "testExtUser",
-		conf.AMAccessControlExternalGroups:    "testExtGroup",
+		conf.AMAccessControlSystemUsers:       "^system:serviceaccount:kube-system:job-controller",
```
Comment (craigcondit) [inline pkg/plugin/admissioncontrollers/webhook/admission_controller_test.go:367]: This should have a trailing '$'

**Review by craigcondit — COMMENTED** [review]
One typo, otherwise looks good.

File: pkg/plugin/admissioncontrollers/webhook/admission_controller_test.go (hunk 1 above)
Comment (pbacsko) [reply pkg/plugin/admissioncontrollers/webhook/admission_controller_test.go]: Done

**Review by craigcondit — APPROVED** [review]
+1 will merge shortly.

**Pull request closed without merging**